src/main/java/edu/iis/powp/adapter/MyAdapter.java

```diff
@@ -9,6 +10,9 @@
  * Plotter adapter to drawer with several bugs.
  */
 public class MyAdapter extends DrawPanelController implements IPlotter {
+
+	DrawPanelController drawerController = DrawerFeature.getDrawerController();

```

Comment:
PLEASE FIX: Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.